Mage.Sets/src/mage/cards/s/SpoilsOfBlood.java

```diff
@@ -26,11 +22,10 @@
 public final class SpoilsOfBlood extends CardImpl {
 
     public SpoilsOfBlood(UUID ownerId, CardSetInfo setInfo) {
-        super(ownerId, setInfo, new CardType[]{CardType.INSTANT}, "{B}");
+        super(ownerId, setInfo, new CardType[] { CardType.INSTANT }, "{B}");
 
         // Create an X/X black Horror creature token, where X is the number of creatures that died this turn.
         this.getSpellAbility().addEffect(new SpoilsOfBloodEffect());
-        this.getSpellAbility().addWatcher(new CreaturesDiedThisTurnWatcher());
```

Comment:
this is necessary for the value to return a non-zero value. Please undo that line deletion.